qa/automation/CareerPortal.Tests.Scenarios/Filters Scenarious.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Filters for vacancies
+
+Background:
+Given User is on https://career.quantori.com/ru/positions
+And vacancies available with Security and Compliance Competence
+And vanacies available with Linux Techology
+
+Scenario: Drop-down of Competence and Technologies
+When User clicks the fields Competence or Technologies
+Then User sees drop-down menu with available Competencies or Technologies
+
+Scenario: Counter for filters
+When User chooses one or more positions in drop-down menu or unchecks the box
+Then Counter in brackets changes its state (increases or decreases)
+
+Scenario: Display of vacancies by the selected Competence
+When User chooses Security and Compliance competence
+Then User sees list of vacancies ( Information Security Specialist) with parameter Remote/Office
```

Comment:
There is only one vacancy for Security and Compliance so I would recommend mention only that one.